conf/test_full.config

```diff
@@ -1,22 +1,24 @@
 /*
- * -------------------------------------------------
- *  Nextflow config file for running full-size tests
- * -------------------------------------------------
- * Defines bundled input files and everything required
- * to run a full size pipeline test. Use as follows:
- *   nextflow run nf-core/circrna -profile test_full,<docker/singularity>
- */
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running full-size tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a full size pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/circrna -profile test_full,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
 
 params {
-  config_profile_name = 'Full test profile'
-  config_profile_description = 'Full test dataset to check pipeline function'
+    config_profile_name        = 'Full test profile'
+    config_profile_description = 'Full test dataset to check pipeline function'
+
+    // Input data for full size test
+    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
+    // TODO nf-core: Give any required params for the test so that command line flags are not needed
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_full_illumina_amplicon.csv'
```

Comment:
is this the right input?

Reply:
nope, I was hoping for some help in getting my full test dataset up and running.